tagsets/helpers.php

```diff
@@ -136,6 +136,28 @@ function ortime__array_ampm_time_to_array_mil_time ($a) { // unused?
     return $r;
 }
 
+function ortime__get_weekday($unixtime,$language='') {
+    global $lang;
+    if (!$language) {
+        if (isset($lang['lang']) && $lang['lang']) {
+            $language=$lang['lang'];
+        } else {
+            global $expadmindata, $settings;
+            if (isset($expadmindata['language']) && $expadmindata['language']) $language=$expadmindata['language'];
+            else $language=$settings['public_standard_language'];
```

Comment:
OK, since you started using using brackets for control statements in this change, it worth using them here as well for consistency.